src/App.tsx

```diff
@@ -1,14 +1,31 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [activeTodo, setActiveTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isOpenModal, setIsOpenModal] = useState(false);
+  const [sortBy, setSortBy] = useState('all');
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    setIsLoading(true);
+    getTodos().then((data) => {
+      setTodos(data);
+      setIsLoading(false);
+    });
+  }, []);
```

Comment:
Consider wrapping your `useEffect` cleanup function with a try-catch block to handle any errors that may occur during the API call or state update. This will provide a better user experience, especially for error handling. 🛠️